### ISSUE TITLE
name-space-charms

### ISSUE THREAD

**Issue by mitechie** [juju/docs#708]
https://jujucharms.com/docs/stable/authors-charm-store#name-space-charms

does not mention the charm store requirements like proof passing. It should be noted so that it's clear that even for namespace charms ingestion won't occur without passing charm proof. (item 4 in the list above)

**Comment by mbruzek** [comment]
@mitechie I agree that the document needs more information, but as we discussed a _charm_ failing proof will still get ingested, only a _bundle_ that fails proof will halt ingestion.
